src/controller/python/chip/CertificateAuthority.py

```diff
@@ -53,7 +53,7 @@ def _Handle(cls):
         return chip.native.GetLibraryHandle()
 
     @classmethod
-    def Logger(cls):
+    def logger(cls):
```

Comment:
Why is this a class method?

If you need a logger for the file and it's not OK to use a property, then you can have a top-level variable in the file that is "logger" for the correct logger and it won't be global.

Reply:
We need a logger instance per class (since the name of each class is different). Having it as a property isn't quite the same, since it's not meant to be instance-specific (i.e it's a static member).

Reply:
Also, having module loggers seems to be a generally frowned upon practice, since due to the way Python parses modules, there's a strong chance any logging configuration that gets applied at runtime won't be seen (see [this](https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/)).

Reply:
Seems the file name and the class name are both `CertificateAuthority`, so I guess we can use

```
_logger = logging.getLogger('CertificateAuthority')
```

in this file and then use _logger in the whole class?

Also, we can use a property for the class:

```
class CertificateAuthority:
    logger = logging.getLogger('CertificateAuthority')
```

The scope of logger will be the class itself instead of a single instance.

Reply:
There are multiple classes in this file, so having a logger at the top won't quite do.

Given what I read [here](https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/), I think the best path forward might just be to make it a member.

Reply:
Will do it a follow-up PR.